goal_src/jak2/engine/target/logic-target.gc

```diff
@@ -677,11 +681,11 @@
                       )
                   )
              )
-    (if (>= (- (-> self clock frame-counter) (-> self control unknown-time-frame21)) (seconds 0.1))
-        (set! (-> self control unknown-time-frame22) (-> self clock frame-counter))
+    (if (>= (- (-> self clock frame-counter) (-> self control unknown-time-frame22)) (seconds 0.1))
+        (set! (-> self control unknown-time-frame23) (-> self clock frame-counter))
```

Comment:
I'd rather not rename fields like this, it makes it very easy to leave behind one that uses the wrong name, and it would be really hard to catch it later.

Reply:
There's not much i can do here, I didn't change any of the types of `control-info` in this PR, I just re-generated the code for the file

Reply:
hm, that's very scary then - we should try to figure out what happened.

Reply:
ok yeah this change is correct. In the future lets be careful about renaming stuff. This file is currently wrong on master.